### PR TITLE
words: where do penguins go when they lose their tail?

### PR DESCRIPTION
**Pull request by catzkorn**
To the retail store.

Signed-off-by: Charlotte Brandhorst-Satzkorn <charlotte@tailscale.com>

### CODE REVIEW

File: words/tails.txt [hunk 1]
```diff
@@ -317,3 +317,19 @@ soay
 xalda
 ewe
 lamb
+great
+emperor
+king
+adelie
+chinstrap
+gentoo
```
Comment (andrew-d) [inline words/tails.txt:325]: Somehow I've made it this many years before discovering that "gentoo" isn't just a Linux distro but also a kind of penguin 🤦‍♂️

File: words/scales.txt [hunk 2]
```diff
@@ -179,3 +179,18 @@ beaver
 hops
 salak
 mercat
+great
```
Comment (andrew-d) [inline words/scales.txt:182]: I don't think penguins have scales?

Comment (catzkorn) [reply words/scales.txt:182]: king-penguin-feet-antarctica-philippe-tulula-and-julie-alexander

Comment (catzkorn) [reply words/scales.txt:182]: > Do Penguins Have Scales?

Every part of a penguin’s body, from its head to its feet, is coated in scales and feathers.

Comment (andrew-d) [reply words/scales.txt:182]: 🤯

**Review by andrew-d — APPROVED** [review]

File: words/scales.txt (hunk 2 above)
Comment (andrew-d) [reply words/scales.txt:182]: I'm also very happy that this means that `great-great` is now a valid tailnet name 🤣

Comment (catzkorn) [reply words/scales.txt:182]: great-gentoo, linux fans are gonna go NUTS

File: words/tails.txt (hunk 1 above)
Comment (shayne) [reply words/tails.txt:325]: EverythingIsConnectedNeilDegrasseTysonGIF

**Pull request merged**